tests/system/feature_store/test_feature_store.py

```diff
@@ -3874,6 +3874,22 @@ def test_ingest_with_validator_from_uri(self, engine):
 
         assert isinstance(df, pd.DataFrame)
 
+    def test_ingest_with_steps_drop_features(self):
+        key = "patient_id"
+        measurements = fstore.FeatureSet(
+            "measurements", entities=[Entity(key)], timestamp_key="timestamp"
+        )
+        measurements.graph.to(DropFeatures(features=[key]))
+        source = CSVSource(
+            "mycsv", path=os.path.relpath(str(self.assets_path / "testdata.csv"))
+        )
+        with pytest.raises(mlrun.errors.MLRunInvalidArgumentError) as ml_run_exception:
+            fstore.ingest(measurements, source)
+        assert (
+            str(ml_run_exception.value)
+            == "DropFeatures can only drop features, not entities"
+        )
```

Comment:
```suggestion
        with pytest.raises(mlrun.errors.MLRunInvalidArgumentError, match="DropFeatures can only drop features, not entities"):
            fstore.ingest(measurements, source)
```
See [here](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions).

Reply:
thank you!
because match runs according to regex , I suggest this change:

match="^DropFeatures can only drop features, not entities$"